<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>461</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wynik losowania</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drawnIndicator1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drawnIndicator2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drawnIndicator3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="drawnIndicator4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>964</x>
      <y>500</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ZAPISZ</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="numberEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="axisAEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>340</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="axisBEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>380</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="coefficientEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>420</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="centerEdit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ustawienia parametrów kulek</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer kulki (1-20)</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Półoś a</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Półoś b</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Współczynnik odbicia</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Przesunięcie śr. ciężkości [%]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
